index.html

```diff
@@ -412,12 +413,80 @@ <h1>Documentation</h1>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu5/apu5_v4.13.0.4.rom">apu5 v4.13.0.4</a><a href="https://3mdeb.com/open-source-firmware/pcengines/apu5/apu5_v4.13.0.4.SHA256" class="sha-button">SHA256</a><a href="https://3mdeb.com/open-source-firmware/pcengines/apu5/apu5_v4.13.0.4.SHA256.sig" class="sha-button">SHA256.sig</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu6/apu6_v4.13.0.4.rom">apu6 v4.13.0.4</a><a href="https://3mdeb.com/open-source-firmware/pcengines/apu6/apu6_v4.13.0.4.SHA256" class="sha-button">SHA256</a><a href="https://3mdeb.com/open-source-firmware/pcengines/apu6/apu6_v4.13.0.4.SHA256.sig" class="sha-button">SHA256.sig</a></li>
+                  </ul>
+                  <div class="smaller-margin">
+                      <p>Zip packages:</p>
+                  </div>
+                  <ul>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu1/apu1_v4.13.0.4.zip">apu1 v4.13.0.4.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu2/apu2_v4.13.0.4.zip">apu2 v4.13.0.4.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu3/apu3_v4.13.0.4.zip">apu3 v4.13.0.4.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu4/apu4_v4.13.0.4.zip">apu4 v4.13.0.4.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu5/apu5_v4.13.0.4.zip">apu5 v4.13.0.4.zip</a></li>
+                    <li><a class="sha-button" href="https://3mdeb.com/open-source-firmware/pcengines/apu6/apu6_v4.13.0.4.zip">apu6 v4.13.0.4.zip</a></li>
+                  </ul>
+                  <p>See an example how to verify the signatures on <a href="https://asciinema.org/a/376207">asciinema</a></p>
+              </div>
+              <div class="source-code-column">
+                  <div class="smaller-margin">
+                      <p>Source code:</p>
+                  </div>
+                  <ul>
+                    <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.13.0.3...v4.13.0.4">coreboot v4.13.0.4</a></li>
+                      <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.12.1.3...rel-1.12.1.3">SeaBIOS rel-1.12.1.3</a></li>
+                      <li><a target="_blank" href="https://github.com/pcengines/sortbootorder/compare/v4.6.19...v4.6.20">sortbootorder v4.6.20</a></li>
+                      <li><a target="_blank" href="https://git.ipxe.org/ipxe.git">ipxe</a></li>
```

Comment:
@Sakoram Could you specify which ipxe version was used? There is no proper versioning there, so let's use commit. It is probably this one `ebf2eaf515e46abd43bc798e7e4ba77bfe529218` but please double check it. So I would expect here a short commit hash in a string shown on website and a link to the specific commit in `a href`.